backend/src/routers/user_auth/get_token.py

```diff
@@ -0,0 +1,36 @@
+from datetime import timedelta
+from fastapi import APIRouter, HTTPException
+from fastapi.encoders import jsonable_encoder
+from starlette.status import HTTP_403_FORBIDDEN
+from starlette.requests import Request
+from starlette.responses import Response
+from .utils import (
+    verify_refresh_token,
+    ACCES_TOKEN_EXPIRES_MINUTES,
+    create_token,
+)
+
+
+router = APIRouter()
+
+
+@router.get("/get-acces-token")
+async def get_acces_token(request: Request):
+    refresh_token = request.cookies.get("refresh_token")
+    response = Response()
+    if refresh_token is None:
+        raise HTTPException(
+            status_code=HTTP_403_FORBIDDEN, detail="No rights to refresh token.",
+        )
+    username = verify_refresh_token(refresh_token)
+    if username:
+        access_token_expires = timedelta(minutes=ACCES_TOKEN_EXPIRES_MINUTES)
+        acces_token = create_token(
+            data={"sub": username}, expires_delta=access_token_expires
+        )
+        acc_token = jsonable_encoder(acces_token)
+        response.set_cookie(
+            key="access_token", value=f"{acc_token}", httponly=True,
```

Comment:
`value=f"{acc_token}"` this looks weird. Did you forget something or do you want to cast the `acc_token` to a string? If you wanted to do the latter `str(acc_token)` should work fine.

Reply:
Also, I think that you want to set the expiration time of the cookie

Reply:
You're right, also according to [jsonable_encoder docs](https://fastapi.tiangolo.com/tutorial/encoder/) i can just do `value=acc_token`. Change coming in next commit.